smother/control.py

```diff
@@ -119,9 +131,25 @@ def load(cls, file_or_path):
         result.data = data
         return result
 
+    def _normalize_path(self, path):
+        aliases = None
+        if self.coverage and self.coverage.config.paths:
+            aliases = PathAliases()
```

Comment:
We should probably only build a PathAliases() object once, when we write the coverage report to disk. It involves compiling a bunch of regexes.